Assignment/solution_student/CSS/portofolio.css

```diff
@@ -0,0 +1,19 @@
+body{
```

Comment:
I would avoid separating the styles into two different stylesheets, especially in a site as small as this one — it's easier to maintain global styles if all styles are in one document, so from now on, organize your styles into one file.